src/lib/slider/slider.ts

```diff
@@ -425,8 +440,10 @@ export class SliderRenderer {
    * Draws ticks onto the tick container.
    */
   drawTicks(tickSeparation: number) {
+    let sliderTrackContainer =
+        <HTMLElement>this._sliderElement.querySelector('.md-slider-track-container');
+    let tickContainerWidth = sliderTrackContainer.getBoundingClientRect().width;
```

Comment:
It was checking the width of the same container whose width it was setting. This wasn't a problem before since it never changed after initialization, but now it can change.